docs/docs/getting-started/first-steps-local.md

```diff
@@ -4,19 +4,25 @@
 With the `constellation mini` command, you can deploy and test Constellation locally without a cloud subscription. This mode is called MiniConstellation. Conceptually, MiniConstellation is similar to [MicroK8s](https://microk8s.io/), [K3s](https://k3s.io/), and [minikube](https://minikube.sigs.k8s.io/docs/).
 <!-- vale on -->
 
-MiniConstellation uses virtualization to create a local cluster with one control-plane node and one worker node. 
+MiniConstellation uses virtualization to create a local cluster with one control-plane node and one worker node.
 
 :::info
 
-MiniConstellation **doesn't** require hardware with Confidential VM (CVM) support. 
+MiniConstellation **doesn't** require hardware with Confidential VM (CVM) support.
 For attestation, MiniConstellation currently uses the software-based vTPM provided by KVM/QEMU.
```

Comment:
Now there are three boxes in a row. I think we can "unbox" the first one and append it to the paragraph above (replacing *MiniConstellation* with *it*). What do you think?

Reply:
I also think this box can be merged into the top paragraph. However I think we should not replace the current top paragraph with it completely, as we would then not inform the user about the virtualization aspect. My suggestion would be something like:
```
MiniConstellation uses virtualization to create a local cluster with one control-plane node and one worker node. It **does not** require hardware with Confidential VM support. For Attestation, it uses software-based vTPM provided by KVM/QEMU.
```

Reply:
Exactly what I had in mind 👍

But note that
* the linter won't allow you to change *doesn't* in *does not*
* it should be "For **a**ttestation, it uses **a** software-based"